src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -874,3 +874,132 @@
     text: >
         az acr check-health --ignore-errors
 """
+
+helps['acr scope-map'] = """
+type: group
+short-summary: Manage scope access maps for Azure Container Registries.
+"""
+
+helps['acr scope-map create'] = """
+type: command
+short-summary: Create a scope map for an Azure Container Registry.
+examples:
+  - name: Create a scope map that allows write and contentRead operations for `hello-world` repository, and pull operations for `hello/world`.
```

Comment:
super nit: Can we call it `hello-world-again` instead of `hello/world` everywhere? Don't like the `/` in the name :)